js/package.json

```diff
@@ -40,5 +40,10 @@
   "packageManager": "pnpm@9.8.0",
   "eslintIgnore": [
     "examples/**/*"
-  ]
+  ],
+  "pnpm": {
+    "overrides": {
+      "@langchain/openai@^0.1>@langchain/core": "^0.1.63"
+    }
+  }
```

Comment:
overrides have to be set at the root, they can't be set in the subpackages, these are both dev dependencies and are necessary to make the tests pass, so should be fine

Reply:
```
 WARN  The field "pnpm.overrides" was found in /Users/parkerstafford/repos/openinference/js/packages/openinference-instrumentation-langchain/package.json. This will not take effect. You should configure "pnpm.overrides" at the root of the workspace instead.

```

Reply:
Got it